src/test/java/hw2/base/Const.java

```diff
@@ -0,0 +1,40 @@
+package hw2.base;
+
+public class Const {
+
+    //names or titles
+    public static final String titleExpected = "Home Page";
+    public static final String login = "Roman";
+    public static final String password = "Jdi1234";
+    public static final String userName = "ROMAN IOVLEV";
```

Comment:
It's not suitable to use constants here because text can be changed and test will fail. Use it directly in test method as assertion data.
Like
assertTextOfSomething(webElementToAssertTextOn,"ROMAN IOVLEV")